api/manage.py

```diff
@@ -0,0 +1,92 @@
+"""This script provides basic utility commands for managing the API"""
+
+import os
+import sys
+
+from scripts import dump_test_data, empty_db_tables, load_test_data
+
+
+def install():
+    """Installs all python requirements"""
+    os.system("pip install -r requirements.txt & pip install -r requirements-dev.txt")
+
+
+def run_linter():
+    """Runs linter"""
+    os.system("pylint mod")
+
+
+def run_tests():
+    """Runs pytest and provides coverage information when '--cov' is provided"""
+    if len(sys.argv) == 3:
+        if sys.argv[2] == "--cov":
+            os.system("pytest mod --cov")
+        elif sys.argv[2] == "--cov-xml":
+            os.system("pytest mod --cov --cov-report=xml")
+        else:
+            print("Wrong parameters")
+    else:
+        os.system("pytest mod")
+
+
+def format():
+    """Formats API as specified from GitHub CI"""
+    os.system("black . & isort .")
+
+
+def build():
+    """Builds API as specified from GitHub CI"""
+    os.system("mypy mod & pylint mod & pytest mod --cov --cov-report=xml")
+
+
+def clear_db():
+    """Clears data rows in database"""
+    empty_db_tables.empty_db()
+
+
+def reset_db():
+    """Clears database entries and loads test data"""
+    clear_db()
+    load_test_data.load()
```

Comment:
I think it would be nice to make one more distinction here (i.e. one more command). One command to just reload the test data and another command to recreate the database (and load data), so delete database, apply migrations, load test data.